solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttpSolrClient.java

```diff
@@ -181,6 +182,7 @@ public HttpClient getHttpClient() {
   public static class Builder extends SolrClientBuilder<Builder> {
     protected final List<String> baseSolrUrls;
     protected HttpSolrClient.Builder httpSolrClientBuilder;
+    private int aliveCheckInterval;
```

Comment:
@epugh this should be init to the default CHECK_INTERVAL same as below https://github.com/apache/solr/pull/1217/files#diff-8a5ab47986f979f1134864e1352f6617e8d425f756018da7e0c1fa191fcc8fafR81